x-pack/plugins/ml/public/application/routing/routes/datavisualizer/data_drift.tsx

```diff
@@ -0,0 +1,107 @@
+} from '../../../datavisualizer/data_drift/index_patterns_picker';
+import { NavigateToPath } from '../../../contexts/kibana';
+import { MlRoute } from '../..';
+import { createPath, PageLoader, PageProps } from '../../router';
+import { ML_PAGES } from '../../../../../common/constants/locator';
+import {
+  breadcrumbOnClickFactory,
+  DATA_COMPARISON_BREADCRUMB,
+  DATA_VISUALIZER_BREADCRUMB,
+  getBreadcrumbWithUrlForApp,
+} from '../../breadcrumbs';
+import { useRouteResolver } from '../../use_resolver';
+import { basicResolvers } from '../../resolvers';
+import { DataSourceContextProvider } from '../../../contexts/ml';
+
+export const dataDriftRouteIndexOrSearchFactory = (
+  navigateToPath: NavigateToPath,
+  basePath: string
+): MlRoute => ({
+  id: 'data_drift',
+  path: createPath(ML_PAGES.DATA_DRIFT_INDEX_SELECT),
+  title: i18n.translate('xpack.ml.dataVisualizer.dataDrift.docTitle', {
+    defaultMessage: 'Data Drift',
+  }),
+  render: (props, deps) => (
+    <PageWrapper {...props} deps={deps} mode={ML_PAGES.DATA_DRIFT_INDEX_SELECT} />
+  ),
+  breadcrumbs: [
+    getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
+    {
+      text: DATA_VISUALIZER_BREADCRUMB.text,
+      ...(navigateToPath
+        ? {
+            href: `${basePath}/app/ml${DATA_COMPARISON_BREADCRUMB.href}`,
+            onClick: breadcrumbOnClickFactory(DATA_COMPARISON_BREADCRUMB.href, navigateToPath),
+          }
+        : {}),
+    },
+    {
+      text: i18n.translate('xpack.ml.trainedModelsBreadcrumbs.dataDriftLabel', {
+        defaultMessage: 'Data Drift',
+      }),
+    },
+  ],
+  'data-test-subj': 'mlPageDataDrift',
+});
+
+export const dataDriftRouteIndexPatternFactory = (
+  navigateToPath: NavigateToPath,
+  basePath: string
+): MlRoute => ({
+  id: 'data_drift',
+  path: createPath(ML_PAGES.DATA_DRIFT_CUSTOM),
+  title: i18n.translate('xpack.ml.dataVisualizer.dataDriftCustomIndexPatterns.docTitle', {
+    defaultMessage: 'Data Drift Custom Index Patterns',
+  }),
+  render: (props, deps) => <PageWrapper {...props} deps={deps} mode={ML_PAGES.DATA_DRIFT_CUSTOM} />,
+  breadcrumbs: [
+    getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
+    {
+      text: DATA_VISUALIZER_BREADCRUMB.text,
+      ...(navigateToPath
+        ? {
+            href: `${basePath}/app/ml${DATA_COMPARISON_BREADCRUMB.href}`,
+            onClick: breadcrumbOnClickFactory(DATA_COMPARISON_BREADCRUMB.href, navigateToPath),
+          }
+        : {}),
+    },
+    {
+      text: i18n.translate('xpack.ml.trainedModelsBreadcrumbs.dataDriftLabel', {
+        defaultMessage: 'Data Drift',
+      }),
+    },
+  ],
+  'data-test-subj': 'mlPageDataDriftCustomIndexPatterns',
+});
+
+interface DataDriftPageProps extends PageProps {
+  mode: 'data_drift_index_select' | 'data_drift_custom';
+}
+const PageWrapper: FC<DataDriftPageProps> = ({ mode }) => {
+  const { context } = useRouteResolver('basic', [], basicResolvers());
```

Comment:
❓ is it actually available for the basic license?

Reply:
Updated her [`08a696a` (#162853)](https://github.com/elastic/kibana/pull/162853/commits/08a696a49eb40f73bb3f78e53a88a38033a11128)